x-pack/plugins/infra/public/pages/metrics/layouts/container.ts

```diff
@@ -54,7 +67,9 @@ export const containerLayoutCreator: InfraMetricLayoutCreator = theme => [
       },
       {
         id: InfraMetric.containerCpuUsage,
-        label: 'CPU Usage',
+        label: i18n.translate('xpack.infra.containerMetrics.cpuUsageSectionLabel', {
```

Comment:
To match the naming scheme from above, how about `xpack.infra.metricDetailPage.containerMetricsLayout.cpuUsageSection.sectionLabel`?